src/app/common/validators/password-match.validator.ts

```diff
@@ -0,0 +1,12 @@
+import {FormGroup} from "@angular/forms";
+
+export const PasswordMatchValidator = (): any => {
+  return (fg: FormGroup) => {
+    const password = fg.get('password')?.value;
+    const repeatPassword = fg.get('repeatPassword')?.value;
+    const error = password === repeatPassword ? null : {repeatPassword: true};
+    fg.get('repeatPassword')?.setErrors(error)
```

Comment:
You are duplicating this: `fg.get('repeatPassword')`

Instead, you can create a variable and reuse it:
```
const repeatPasswordControl = fg.get('repeatPassword');
Line 6: repeatPasswordControl?.value
Line 8: repeatPasswordControl?.setErrors(error)
```